common/lib/xmodule/xmodule/js/src/capa/display.coffee

```diff
@@ -443,14 +445,20 @@ class @Problem
     if not @submit_save_waitfor(@save_internal)
       @disableAllButtonsWhileRunning @save_internal, false
 
+  focus_on_save_notification: =>
+    if @saveNotificationArea.length > 0
+      @saveNotificationArea.focus()
+
   save_internal: =>
     Logger.log 'problem_save', @answers
     $.postWithPrefix "#{@url}/problem_save", @answers, (response) =>
       saveMessage = response.msg
       if response.success
         @el.trigger('contentChanged', [@id, response.html])
-      @gentle_alert saveMessage
-      @updateProgress response
+        @render(response.html, @focus_on_save_notification)
+        @updateProgress response
+      else
+        @gentle_alert saveMessage
```

Comment:
Yes I tested the error messages.  They are still posted when sent.

I think we should wait to add test coverage for the story when we refactor this message type.